common/static/common/templates/discussion/new-post.underscore

```diff
@@ -29,6 +29,13 @@
             </div>
         </label>
     </div>
+    <% } else if (is_discussion_division_enabled) { %>
+    <div class="post-field group-visibility">
+        <label class="field-label">
```

Comment:
I think ```<label>``` is inappropriate here.  ```<label>``` elements are reserved to identify form input and this doesn't appear to be the label of any form input.  I do think that it should be programmatically linked to a form input so that screen reader users are aware what the visibility of the post is while they are completing the form (they are usually not made aware of text that appears between form inputs in a form.)  Can we just make this a div with an ID attribute and then link it to "Topic Area" select menu via aria-describedby?  This select already has an aria-describedby attribute that references "field_help_visible_to".  aria-describedby takes a space separated list of IDs, so just append the ID for this on to the end.

Reply:
@cptvitamin Thanks for the suggestion; I've changed this accordingly.  I have one more question:  the visibility message is only shown when the discussions are actually divided.  Is it still possible to add the id unconditionally to the `aria-describedby` attribute?  If the visibility message is not shown, the id will be invalid.  Is this a problem?
  I would be able to only conditionally add the id, but this requires some further changes, and I would only want to do them if they are actually needed.

Reply:
Hi @smarnach, yes that would be a problem.  All IDrefs must actually exist in the DOM. The nodes can be empty, but the nodes do need to exist.  Since you can have a zero height div, it's recommended to always have containers available in the DOM on page load and only make their content conditionally visible, not making the inclusion of the node conditional.  This is especially important for any container that includes an aria-live attribute, which the screen reader will monitor for changing content.  If the container is added dynamically, the screen reader will not have marked it for monitoring, thus updates will not be observed and reported to the user.

Reply:
@cptvitamin Thanks for the guidance, I made sure the div is always included, and verified that it renders conrrectly both for divided and undivided discussions.